src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -508,7 +508,7 @@ Use the dropdown to view and switch to other projects this file may belong to.</
     <value>Synchronizing with {0}...</value>
   </data>
   <data name="FullSolutionAnalysisOff" xml:space="preserve">
-    <value>Low memory detected. Full solution analysis disabled for this solution.</value>
+    <value>We have suspended some advanced features to improve performance.</value>
```

Comment:
+1. Given that the current change only turns off analysis for current solution instance, i.e. it will turn it back on when user closes and reopens the solution, we should either make this message or the README document clearly state that the advanced features are being turned off for _the current solution for current VS instance_.